api-server/README.md

```diff
@@ -0,0 +1,112 @@
+6. There's no volume set. Meaning: Once the container is stopped and removed, all the data in the database will be lost. Please consult the documentation of the container to learn how to preserve the database data.
+
+Please understand that this is just a minimal example, and for real infrastructure, proper security must be considered.
+
+### How to run the blockchain scanner daemon
+
+Assuming the database works as described earlier, the blockchain scanner daemon can be run from the source code using:
+
+```
+cargo run --bin api-blockchain-scanner-daemon --release -- --network testnet
+```
+
+or if you want to run using the executable directly:
+
+```
+api-blockchain-scanner-daemon --network testnet
+```
+
+And this should immediately work, where the blockchain scanner daemon will communicate with the default network RPC network port of the node (13030 for testnet, 3030 for mainnet), and it will also communicate with the database and write the data it finds in the blockchain.
+
+If you need to configure extra options, such as postgres username and password, just add `--help` to the commands above, and the commands will be shown.
+
+### How to run the API web server
+
+After having filled the database with information, the API web server can use this information to respond to http requests, whether for requests from the public, or your internal infrastructure for other purposes.
+
+Assuming the database server is setup on the same machine and is reachable with `127.0.0.1:5432`, you can use the following command to run the API web server, compiled from the source code:
+
+```
+cargo run --bin api-web-server --release -- --network testnet --bind-address 127.0.0.1:3000
+```
+
+Or, you can just run the executable if you have:
+
+```
+api-web-server --network testnet --bind-address 127.0.0.1:3000
+```
+
+The API web server will immediately start and connect to the database locally. A specific remote database can be specified using command line arguments. Add `--help` to the previously mentioned commands to see how to do this.
+
+### Testing the API web server
+
+Once the previous steps are complete, you're ready to communicate with the API web server. The following curl command should work (or you can put the link in your browser directly):
+
+```
+curl http://127.0.0.1:3000/api/v1/chain/tip
+```
+
+which will return the best block information in the blockchain. Or:
+
+```
+curl http://127.0.0.1:3000/api/v1/chain/10
+```
+
+to get the id of the block at height 10.
+
+Make sure the scanner is fully synced to get correct information about the current state of the blockchain.
+
+### Logging
+
+The same logging rules [in the main readme file](/README.md) apply here as well. By default, all the programs we use have INFO level logging.
```

Comment:
> By default, all the programs we use have INFO level logging.

Probably it'd be better to say something like "By default, all our programs use INFO level logging."